scripts/CMSRunAnalysis.py

```diff
@@ -696,8 +674,8 @@ def getOutputModules():
         with tempSetLogLevel(logger=logging.getLogger(), level=logging.ERROR):
             ret = scram("python -c '%s'" % pythonScript, runtimeDir=os.getcwd())
         if ret > 0:
-            msg = scram.diagnostic()
-            handleException("FAILED", EC_CMSRunWrapper, 'Error getting output modules from the pset.\n\tScram Env %s\n\tCommand:%s' % (msg, pythonScript))
```

Comment:
The `pythonScript` that was passed [here](https://github.com/dmwm/CRABServer/pull/5935/files#diff-b4df3d7055437f8d2eeafa0b449218d5L700) will be incluced in the  `scram.diagnostic()` [here](https://github.com/dmwm/CRABServer/pull/5935/files#diff-b4df3d7055437f8d2eeafa0b449218d5R677)?
If not we loose this information.

Reply:
good point. scram.diagnostic() reports the command, but let me double check that all parameters are there etc.

Reply:
yes. the output from `scram.diagnostic()` is
```
        Scram Command Diagnostic:
        Command : scramv1
        Architecture: slc6_amd64_gcc700
        Executed: python -c 'from PSetTweaks.WMTweak import makeTweak;config = __import__("WMTaskSpace.cmsRun.PSet", globals(), locals(), ["process"], -1);tweakJson = makeTweak(config.process).jsondictionary();print tweakJson["process"]["outputModules_"]'
        Exit Status: 0
        Stdout: ['out']
```